deployer/orb.yaml

```diff
@@ -40,7 +40,9 @@ jobs:
           command: |
             if [[ ! -z "<<parameters.verify_version>>" ]]; then
                 code_version=$(find "<<parameters.folder>>/dist" -iname '*.tar.gz' | awk -F- '{sub(/\.tar\.gz/, ""); print $NF}')
-                if [[ <<parameters.verify_version>> != ${code_version} ]]; then
+                code_version_alt="${code_version/.dev/-dev.}"
```

Comment:
Is there a reason we can't enforce that git tags match the code versions? Can we just have our git tags enforced to be `*-dev.*` instead of `*.dev*`?

Reply:
its because the tarball that gets generated is of the format `.dev\d+` but semver generates it a git tag of `-dev.\d+`